nx-X11/programs/Xserver/os/io.c

```diff
@@ -965,6 +965,10 @@ WriteToClient (ClientPtr who, int count, const void *__buf)
 	  CriticalOutputPending = FALSE;
 	  NewOutputPending = FALSE;
 	}
+
+	if (FlushCallback)
+	    CallCallbacks(&FlushCallback, NULL);
+
 	return FlushClient(who, oc, buf, count);
```

Comment:
@Ionic: this part of the commit now becomes obsolete. Has been moved to FlushClient() function later in 2016.